pkg/bpf/testdata/unreachable-tailcall.c

```diff
@@ -0,0 +1,144 @@
+typedef unsigned int __u32;
+
+enum bpf_map_type {
+	BPF_MAP_TYPE_UNSPEC,
+	BPF_MAP_TYPE_HASH,
+	BPF_MAP_TYPE_ARRAY,
+	BPF_MAP_TYPE_PROG_ARRAY,
+};
+
+struct __sk_buff {
+	__u32 len;
+	__u32 pkt_type;
+	__u32 mark;
+	__u32 queue_mapping;
+	__u32 protocol;
+	__u32 vlan_present;
+	__u32 vlan_tci;
+	__u32 vlan_proto;
+	__u32 priority;
+	__u32 ingress_ifindex;
+	__u32 ifindex;
+	__u32 tc_index;
+	__u32 cb[5];
+	__u32 hash;
+	__u32 tc_classid;
+	__u32 data;
+	__u32 data_end;
+	__u32 napi_id;
+};
+
+
+struct bpf_elf_map {
```

Comment:
Can we import these from actual headers used in the datapath instead? (also the other macros below)

Reply:
Hmm, let me try. I think so if we add something like a `-I $(PROJECT_ROOT)/bpf/lib` to the clang invocation